projects/storefrontapp-e2e-cypress/cypress/integration/mobile/my-account/address-book-mobile.e2e-spec.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  addressBookTest,
+  verifyAsAnonymous,
+} from '../../../helpers/address-book';
+import { formats } from '../../../sample-data/viewports';
+import * as login from '../../../helpers/login';
+import * as homepage from '../../../helpers/homepage';
+
+describe(`${formats.mobile.width + 1}p resolution - Address Book page`, () => {
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+  });
+
+  beforeEach(() => {
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+  });
+
+  describe('address book test for anonymous user', () => {
+    verifyAsAnonymous();
+  });
+
+  describe('address book test for logged in user', () => {
+    before(() => {
+      cy.requireLoggedIn();
+      cy.reload();
+      cy.visit('/');
+    });
+
+    beforeEach(() => {
+      cy.restoreLocalStorage();
+      homepage.clickHamburger();
+      cy.selectUserMenuOption({
```

Comment:
we need to move this on the before as the last statement because right now the test is trying to always access the page when it is already on that page. Also, the double hamburger click is making it close the menu

Reply:
cannot do this in all the tests, for eg, the close account test, there's a test that checks if we click on cancel, we go to homepage. so we're not on close-account page. 
I'll see how many tests we have like this, either I make it go to that page explicitly in the tests, or will leave them in beforeEach for these ones.

Reply:
Sorry! I was wrong about moving it all, some need to be inside the before and some needs to be in the beforeEach 👍 But yeah the double hamburger one was good to fix